src/test/skript/tests/syntaxes/conditions/CondIsDivisibleBy.sk

```diff
@@ -0,0 +1,6 @@
+test "is divisible":
+	assert 5 can be divided by 5 with "5 can be divided by 5!"
```

Comment:
I am guessing you forgot to update this when you added the `evenly` to the syntax

Reply:
yeah oops